libs/labelbox/src/labelbox/client.py

```diff
@@ -563,14 +565,16 @@ def get_labeling_frontends(self, where=None) -> List[LabelingFrontend]:
         """
         return self._get_all(Entity.LabelingFrontend, where)
 
-    def _create(self, db_object_type, data):
+    def _create(self, db_object_type, data, extra_params={}):
```

Comment:
Could this extra_params arg just be **kwargs?

Reply:
I understand that kwargs typically allows for passing any general pair of name and value parameters in bulk. However, in this instance, I specifically need a list of GraphQL filters. If I use kwargs here, I would need to structure my data as {extra_params: {...}}, which seems unnecessary. What do you think?

Reply:
Ok yeah, it's fine as is